django/contrib/admin/views/main.py

```diff
@@ -35,10 +37,19 @@
     ALL_VAR, ORDER_VAR, ORDER_TYPE_VAR, SEARCH_VAR, IS_POPUP_VAR, TO_FIELD_VAR)
 
 
+class ChangeListSearchForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        super(ChangeListSearchForm, self).__init__(*args, **kwargs)
+        self.fields = {
+            SEARCH_VAR: forms.CharField(required=False, strip=False),
+        }
+
+
 class ChangeList:
     def __init__(self, request, model, list_display, list_display_links,
                  list_filter, date_hierarchy, search_fields, list_select_related,
-                 list_per_page, list_max_show_all, list_editable, model_admin, sortable_by):
+                 list_per_page, list_max_show_all, list_editable, model_admin, sortable_by,
+                 search_form_class=ChangeListSearchForm):
```

Comment:
I meant just as a class attribute, rather than passing it here... Maybe we could do this... (What are your thoughts?)